staging/src/k8s.io/apiserver/pkg/admission/plugin/validatingadmissionpolicy/admission.go

```diff
@@ -175,7 +182,7 @@ func (c *celAdmissionPlugin) Validate(
 		return admission.NewForbidden(a, fmt.Errorf("not yet ready to handle request"))
 	}
 
-	return c.evaluator.Validate(ctx, a, o)
+	return c.evaluator.Validate(ctx, a, o, c.authorizer) // TODO: pass in c.Authorizer?
```

Comment:
TODO asking about exposing `authorizer`?  Why would we expose it?

Reply:
> You might like to sketch out the docs for this _before_ we merge the code change @jpbetz, to make sure that we know what we need to explain.

Good call @sftim , I've added docs for the Kubernetes CEL libraries, including this one, to https://github.com/kubernetes/website/pull/39642. I've attempted to answer the questions you asked there.

Reply:
Oops, this TODO was just to remind me where the dependency injection should go while wiring things up. Removing.